org.eclipse.m2e.rcptt.tests/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.eclipse.m2e.tests.rcptt</name>
```

Comment:
The project Name should Match the folder/Module Name.
```suggestion
	<name>org.eclipse.m2e.rcptt.tests</name>
```

Reply:
Whoops. Done and dusted. :)
I hope the test is at least somewhat understandable... Looking at the test case via a text editor is a lot more difficult than via the IDE.